tsconfig.json

```diff
@@ -18,9 +18,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "noImplicitAny": false,
```

Comment:
I had a problem with import react-router-dom. I use this to solve the problem

Reply:
I wonder if we can solve this issue another way :) Probably we will need this eslint rule set to true for better typescript code.

Reply:
ok thanks, I found another solution;)